### ISSUE TITLE
[Feature Request] Expand automated mobile testing to cover more platforms

### ISSUE THREAD

**Issue by trwalke** [AzureAD/microsoft-authentication-library-for-dotnet#1643]
**Is your feature request related to a problem? Please describe.**
We need to investigate how much coverage we can implement using the current appcenter/Xamarin.UiTest infrastructure so that we can cover more platforms on android and iOS.

**Describe the solution you'd like**
Automated testing coverage on appcenter to include platforms as far back as Android JellyBean (4.1.0+) and iOS 10+. We can keep the large automated testing jobs on the release build only at first. We should also have the tests run in parallel on the various platforms to reduce the overall time of the test run.

**Describe alternatives you've considered**
It is possible to run these scenarios on an emulator for Android and iOS as we can easily create images of older android versions. This will make it easy to run some tests manually. While it is ideal to make the additional test coverage automated, if there are gaps that we cannot cover with automated testing then this may be the next best thing outside of acquiring older devices.

**Comment by jmprieur** [comment]
@henrik-me @bgavrilMS @trwalke @jennyf19 @neha-bhargava 
do we think we'll ever do that? should it be a GitHub issue?

**Comment by bgavrilMS** [comment]
I had a prototype going using Appium. It's doable, but requires a lot of time. Not a priority at the moment I think.

**Comment by henrik-me** [comment]
Not sure the right approach for this expansion is App Center. Closing this one and when ready we can explore how to improve our automated tests especially to include system browser and broker testing.
